jormungandr/src/rest/v0/handlers.rs

```diff
@@ -418,3 +419,34 @@ pub fn get_utxo(context: State<Context>, path_params: Path<(String, u8)>) -> Act
             })
         })
 }
+
+pub fn get_stake_pool(context: State<Context>, pool_id_hex: Path<String>) -> ActixFuture!() {
+    pool_id_hex
+        .parse()
+        .map_err(ErrorBadRequest)
+        .into_future()
+        .and_then(move |pool_id| {
+            chain_tip_fut(&context).and_then(move |blockchain_tip| {
+                let ledger = blockchain_tip.ledger();
+                let vrf_pk = ledger
+                    .delegation()
+                    .lookup(&pool_id)
+                    .ok_or_else(|| {
+                        ErrorNotFound(format!("Stake pool '{}' not found", pool_id_hex))
+                    })?
+                    .keys
+                    .vrf_public_key
+                    .to_bech32_str();
+                let value = ledger
+                    .get_stake_distribution()
+                    .to_pools
+                    .get(&pool_id)
+                    .map(|pool| pool.total.total_stake.0)
+                    .unwrap_or(0);
+                Ok(Json(json!({
+                    "vrfPublicKey": vrf_pk,
+                    "value": value,
```

Comment:
since this is a stake distribution that would be better to keep a clear name: `total_stake` or something similar